llvm-bazel/.bazelrc

```diff
@@ -62,6 +62,48 @@ build:generic_gcc --copt=-Wno-misleading-indentation --host_copt=-Wno-misleading
 # Use `-Werror` for GCC to make sure warnings don't slip past.
 build:generic_gcc --copt=-Werror --host_copt=-Werror
 
+###############################################################################
+# Windows specific flags for building with VC.
+###############################################################################
+
+build:msvc --copt=/WX     # Treat warnings as errors...
+
+# ...but disable the ones that are violated
+build:msvc --copt=/wd4624 # destructor was implicitly defined as deleted
+build:msvc --copt=/wd4244 # possible loss of data
+build:msvc --copt=/wd4005 # macro redefinition
+build:msvc --copt=/wd4141 # inline used more than once
+build:msvc --copt=/wd4267 # initializing: possible loss of data
+build:msvc --copt=/wd4273 # multiple definitions with different dllimport
+build:msvc --copt=/wd4319 # zero-extending after complement
+build:msvc --copt=/wd4804 # comparisons between bool and int
+build:msvc --copt=/wd4805 # comparisons between bool and int
+
+build:msvc --linkopt=/WX # Treat warnings as errors...
+
+# ...but disable the ones that are violated.
+build:msvc --linkopt=/IGNORE:4001 # no object files
+build:msvc --linkopt=/IGNORE:4217 # mismatch import/export declspec
+
+# Enables unix-style runfiles link trees (requires symlink permission).
+# See: https://blogs.msvc.com/msvcdeveloper/2016/12/02/symlinks-msvc-10/
+# Generally: Enable Developer Mode in the Developer Settings page of the
+# system settings.
+build:msvc --experimental_enable_runfiles
```

Comment:
I guess this is something Bazel specific? Should there be a comment about it is? If users need to jump through hoops for this, at least it's nice to know what it's for.

Reply:
I guess I don't know what a symlink runfiles tree is, but maybe I don't need to know :) If it's not important, I'd recommend skipping it though, since requiring symlinks on Windows might not work for everyone.

Reply:
It's a Bazel thing. For sandboxing purposes Bazel will create a symlink tree of files available to an action (e.g. a test) based on its dependencies. The action is then run with only its declared dependencies available. Bazel will use symlinks to create these trees. Let me see whether we can drop this.

Reply:
Works without. Let's leave it off unless @stellaraccident is there a reason we need this in IREE that isn't just "TF uses it"?